samples/multi-update.bolt

```diff
@@ -0,0 +1,22 @@
+// Firebase can update multiple paths with a single update.
```

Comment:
missed that. nice! https://www.firebase.com/blog/2015-09-24-atomic-writes-and-more.html